src/components/molecules/Wallet/Details.tsx

```diff
@@ -26,6 +27,39 @@ export default function Details(): ReactElement {
     setProviderInfo(providerInfo)
   }, [web3Provider])
 
+  async function addOceanToWallet() {
+    const tokenMetadata = {
+      type: 'ERC20',
+      options: {
+        address: config.oceanTokenAddress,
+        symbol: networkId === 137 ? 'mOCEAN' : 'OCEAN',
```

Comment:
this should be in config as well `config.oceanTokenSymbol`, we must work under the premise that we have multiple chains

Reply:
yes this would be nice but I can't access it from `config`

Reply:
fully agree, but not sure we will solve this as part of this PR so the dedicated detection here should be fine for launching it. I also hacked in a bunch of temporary `networkId === 137` checks in web3 branch, also for symbol. So we should first add `config.oceanTokenSymbol` over in ocean.js and after that change all the dedicated network checks. Prepared that PR over on ocean.js https://github.com/oceanprotocol/ocean.js/pull/670